src/torchcodec/_core/CMakeLists.txt

```diff
@@ -144,11 +178,14 @@ function(make_torchcodec_libraries
     # stray initialization of py::objects. The rest of the object code must
     # match. See:
     #   https://pybind11.readthedocs.io/en/stable/faq.html#someclass-declared-with-greater-visibility-than-the-type-of-its-field-someclass-member-wattributes
-    target_compile_options(
-        ${pybind_ops_library_name}
-        PUBLIC
-      "-fvisibility=hidden"
-    )
+    if(NOT WIN32)
+        target_compile_options(
+            ${pybind_ops_library_name}
+            PUBLIC
+            "-fvisibility=hidden"
+        )
+    endif()
```

Comment:
I am not entirely sure that this needs to be skipped on Windows. But the compilation is working so far. This *may* be the reason the tests were failing, I'm not sure. I'll investigate in a follow-up.